integration/tests/cook/test_basic.py

```diff
@@ -898,12 +893,7 @@ def test_memory_limit_exceeded_cook_script(self):
 
     @pytest.mark.memlimit
     def test_memory_limit_exceeded_mesos_script(self):
-        settings_dict = util.settings(self.cook_url)
-        set_memory_limit = settings_dict.get("kubernetes", {}).get("set-memory-limit?", True)
-        if util.using_kubernetes() and not set_memory_limit:
-            command = self.infinite_memory_script_command(count=2048)
-        else:
-            command = self.memory_limit_script_command(count=2048)
+        command = self.memory_limit_script_command(count=2048)
```

Comment:
(this is not about this line, just didn't know where else to put this question)

Are we adding new integration test(s) in this PR or no?